docs/makeup-menu/index.js

```diff
@@ -12,11 +12,19 @@ import Menu from "../../packages/makeup-menu";
 const log = (e) => console.log(e.type, e.detail);
 
 window.onload = function () {
-  document.querySelectorAll(".menu").forEach(function (el, i) {
+  document.querySelectorAll(".menu:not(.menu-icon)").forEach(function (el, i) {
```

Comment:
Same thing here. Keep the same selector and conditionally add the additional `prop:value` pairs based on whether `.menu-icon` is present.